packages/grafana-ui/src/components/Graph/GraphTooltip/MultiModeGraphTooltip.test.tsx

```diff
@@ -92,12 +93,13 @@ describe('MultiModeGraphTooltip', () => {
       );
 
       // We rendered two series rows
-      const rows = container.find('SeriesTableRow');
+      const rows = screen.getAllByTestId('SeriesTableRow');
+      expect(rows.length).toEqual(2);
 
       // We expect A-series(1st row) not to be highlighted
-      expect(rows.get(0).props.isActive).toBeFalsy();
+      expect(rows[0]).not.toHaveStyle('font-weight: 500');
       // We expect B-series(2nd row) not to be highlighted
-      expect(rows.get(1).props.isActive).toBeFalsy();
+      expect(rows[1]).not.toHaveStyle('font-weight: 500');
```

Comment:
**Question:** would we be able to use the theme settings here somehow to make these tests a little bit more robust? (I mean something like `theme.typography.fontWeightMedium`)